app/templates/partials/forms/select.html

```diff
@@ -1,12 +1,17 @@
 <select class='input input--select' id='input-{{select.name}}' name='{{select.name}}'>
+
+  {% if not (select.selected and select.mandatory) %}
+      <option value="">{{select.placeholder}}</option>
+  {% endif %}
+
```

Comment:
Can you check with a designer and Hamish about this behaviour; I thought the placeholder would always be in the list but just not selectable (disabled) for mandatory questions; i.e. should you actually be removing the placeholder?